packages/create/src/generators/wc-lit-element-ts/templates/_tsconfig.json

```diff
@@ -0,0 +1,15 @@
+  {
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "noEmitOnError": true,
+    "lib": ["es2017", "dom"],
+    "strict": true,
+    "esModuleInterop": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "outDir": "out-tsc"
```

Comment:
For the web component this should be `dist` but I guess that creates a problem with the rollup build?